virttest/utils_libvirt/libvirt_vmxml.py

```diff
@@ -21,3 +23,15 @@ def set_vm_attrs(vmxml, vm_attrs):
     vmxml.xmltreefile.write()
     vmxml.sync()
     return vmxml
+
+
+def remove_vm_device_by_type(vm_name, device_type):
```

Comment:
Is there any difference when this method is used on running VM and stopped Vm?

Reply:
updated.